pandas/core/window/rolling.py

```diff
@@ -59,6 +56,81 @@
 from pandas.tseries.offsets import DateOffset
 
 
+def calculate_center_offset(window) -> int:
+    """
+    Calculates an offset necessary to have the window label to be centered
```

Comment:
Sorry to be pedantic :) But should this follow the docstring guide?

> The short summary must start with a capital letter, **end with a dot**, and fit in a single line. It needs to express what the object does without providing details. For functions and methods, the short summary must **start with an infinitive verb**.

same for the `calculate_min_periods` docstring